src/main/io/gps.c

```diff
@@ -841,6 +1322,11 @@ void gpsUpdate(timeUs_t currentTimeUs)
     timeUs_t executeTimeUs;
     gpsState_e gpsCurrentState = gpsData.state;
 
+
+#if DEBUG_SERIAL_BAUD
+    debug[2] = (int16_t)((currentTimeUs - gpsData.lastMessage) / 100);
```

Comment:
```suggestion
    debug[2] = (int16_t)(cmp32(currentTimeUs, gpsData.lastMessage) / 100);
```
`currentTimeUs - gpsData.lastMessage` is unsigned, so this will produce incorrect result if `currentTimeUs < gpsData.lastMessage`

Also, DEBUG_SET ?

Reply:
Also, `gpsData.lastMessage` is millis, comparing to `currentTimeUs` is definitely wrong

Reply:
@adam-ah   I have changed the time comparisons, please check I did that correctly.